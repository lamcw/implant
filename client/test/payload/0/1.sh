#!/bin/sh

./sc-client payload --exec merlin | sed 's/\x1b\[[0-9;]*m//g' > output.txt
diff -Z output.txt client/test/payload/0/1.stdout
