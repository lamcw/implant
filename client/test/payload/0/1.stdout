[*] Merlin extracted successfully
[*] Merlin executed successfully
